en/editor/scene/transform-gizmo.md

```diff
@@ -0,0 +1,60 @@
+# Transform Gizmo
+We mainly use the **Transform Tool** on the toolbar which is in the left top of main window to arrange the node in the scene.
+
+## Move transform tool
+
+**Move transform tool** is the default active transform tool. This tool can be activated by clicking the first button on top left corner of main window's tool bar, or press the keyboard shortcut **W** in scene editor.
+
+![position tool](images/position-tool.png)
+
+After you selected a node, You can see three move handle which is made up of red, green, blue arrow and plane in the center(or the anchor position) of the node.
+
+Gizmo refers to a controller that can interactive with mouse and appears in certain editing states in the scene editor. These controllers are only to assist editing without being shown in the game while it is running.
+
+![position gizmo](images/position-gizmo.png)
+
+When the move transform tool is active：
+- Press red,green,blue arrow and drag mouse, then the node will be moved on the x,y,z axis.
+- Press red,green,blue plane and drag mouse, then the node will be moved freely on y-z,x-z,x-y plane.
+
+## Rotate transform tool
+Click the second button on the tool bar located at the main window's top left corner or press the keyboard shortcut **E** in scene editor and you can activate **Rotate transform tool**.
+
+![rotation tool](images/rotation-tool.png)
+
+The gizmo of rotate transform tool is made up of three orthogonal circles(an arrow and a circle in 2D view).By dragging arrow or any point in the circle, you can rotate the node and you can see the rotation area on gizmo before you release the mouse.
```

Comment:
The gizmo of rotate transform tool is made up of three orthogonal circles(an arrow and a circle in 2D view).By dragging arrow or any point in the circle, you can rotate the node and you can see the rotation area on gizmo before you release the mouse.
-->
The gizmo of rotate transform tool is made up of three orthogonal circles (an arrow and a circle in 2D view). By dragging arrow or any point in the circle, you can rotate the node and you can see the rotation area on gizmo before you release the mouse.